cdap-master/src/main/java/co/cask/cdap/data/tools/Main.java

```diff
@@ -243,12 +243,15 @@ private static void upgradeUserTables(final Injector injector) throws Exception
 
         final boolean supportsIncrement =
           "true".equalsIgnoreCase(desc.getValue(Table.PROPERTY_READLESS_INCREMENT));
+        final boolean transactional =
+          !"true".equalsIgnoreCase(desc.getValue(Constants.Dataset.TABLE_TX_DISABLED));
```

Comment:
if Constants.Dataset.TABLE_TX_DISABLED is true, then its non-transactional right ?